docs/en/stack/ml/df-analytics/flightdata-regression.asciidoc

```diff
@@ -246,33 +282,46 @@ The snippet below shows a part of a document with the annotated results:
           "DestRegion" : "UK",
           "OriginAirportID" : "LHR",
           "DestCityName" : "London",
-          "FlightDelayMin" : 66,      <1>
+          "FlightDelayMin" : 66,
           "ml" : {
-            "FlightDelayMin_prediction" : 62.527,   <2>
-            "is_training" : false   <3>
+            "FlightDelayMin_prediction" : 62.527,
+            "is_training" : false
           }
           ...
 ----
-
-<1> The `dependent_variable` with the ground truth value. This is what we are 
-trying to predict with the {reganalysis}.
-<2> The prediction. The field name is suffixed with `_prediction`.
-<3> Indicates that this document was not used in the training set.
+====
 
 
 [[flightdata-regression-evaluate]]
-==== Evaluating results
-
-The results can be evaluated for documents which contain both the ground truth 
-field and the prediction. In the example below, `FlightDelayMins` contains the 
-ground truth and the prediction is stored as `ml.FlightDelayMin_prediction`.
-
-. Use the {dfanalytics} evaluate API to evaluate the results.
-+
---
-First, we want to know the training error that represents how well the model 
-performed on the training dataset:
-
+==== Evaluating {regression} results
+
+Though you can look at individual results and compare the predicted value
+(`ml.FlightDelayMin_prediction`) to the actual value (`FlightDelayMins`), you
+typically need to evaluate the success of the {regression} model as a whole.
+
+{kib} provides _training error_ metrics, which represent how well the model
+performed on the training dataset. It also provides _generalization error_
+metrics, which represent how well the model performed on testing data.
+
+For more information about the interpreting the evaluation metrics, see
+<<ml-dfanalytics-regression-evaluation>>.
+
+////
+TBD: Can we provide tips for what are good MSE or R-squared values? For example:
+A MSE of zero means that the models predicts the dependent variable with
+perfect accuracy. This is the ideal, but is typically not possible. Likewise, an
+R-squared value of 1 indicates that all of the variance in the dependent variable
+can be predicted from the feature variables. Typically, you compare the MSE and
```

Comment:
"can be predicted from the feature variables" -> "can be explained by the feature variables"